R/individuals.R

```diff
@@ -0,0 +1,66 @@
+# Define population sizes
+human_population <- 100 * 1000
+mosquito_limit <- 100 * human_population
+
+#' @title Define model states
+#' @description
+#' create_states creates the human and mosquito states for the model
+create_states <- function() {
+  list(
+    # Human states
+    S       = individual::State$new("S", human_population),
+    I       = individual::State$new("I", 0),
```

Comment:
Trying to match these with my internal map of states. Is "I" the liverstage, i.e. exposed state that leads on to diseased ("D") or treated ("T"), or have I got this wrong? Also, I can't see a prophylactic state.

Reply:
Yes! I made the "I" (or Infected) the liverstage. It's probably worth me adding documentation to describe what these states and parameters mean. I've made an issue for that [here](https://github.com/mrc-ide/malariasimulation/issues/4)

Reply:
Prophylaxis is a great find. It looks like the later paper models the susceptability directly on humans in the "T" state. I've added a "P" state because I think it could be clearer.